test/Microsoft.TestPlatform.TestUtilities/IntegrationTestBase.cs

```diff
@@ -749,17 +748,40 @@ protected string BuildMultipleAssemblyPath(params string[] assetNames)
             return string.Join(" ", assertFullPaths);
         }
 
+        protected static string GetDiagArg(string rootDir)
```

Comment:
Most of these helpers that don't rely on the class state could potentially be moved to a `TestHelper` or `TestUtils` class as it feels a little weird to see/read `IntegrationTestBase.XXX`.

Reply:
we should have already some inheritance we can add one more on top of `IntegrationTestBase` 
```cs
 public class AcceptanceTestBase : IntegrationTestBase
    {
```

Reply:
I don't want to gold plate this too much. But it seems that the getResultsDirectory and TryRemoveDirectory could become part of the `Workspace` class. And it looks like a good fit especially if we renamed it to TempDirectory, and changed the tests that currently grab a random temp path as a string, and then later try to clean that up (either in test body - which will fail if test fails, or in some cases in cleanup - which we would just replace with a call on a workspace / tempdirectory instance). 

But none of this seems necessary to do in this PR.